functions/java-jre.sh

```diff
@@ -17,7 +17,28 @@ java_webupd8() {
   if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; exit 1; fi
 }
 
-java_zulu_embedded() {
+java_zulu() {
+  FILE="$(mktemp /tmp/.XXXXXXXXXX)"
+
+  if is_arm; then
+    JAVA=zulu8.31.1.122-jdk1.8.0_181-linux_aarch32hf
+  else
+    JAVA=zulu8.31.0.1-jdk8.0.181-linux_x64
+  fi
+  querytext="Downloading Zulu implies to agree to Azul Systems' Terms of Use. Display them now or proceed with downloading."
+  while ! (whiptail --title "Download Zulu Java" --yes-button "Download and Install" --no-button "Read Terms of Use" --defaultno --yesno "$querytext" 10 80) ; do
+    whiptail --textbox /opt/openhabian/docs/azul-zulu-license.md --scrolltext 27 116
+  done
+
+  wget -nv -O $FILE http://cdn.azul.com/zulu-embedded/bin/${JAVA}.tar.gz
+  cd /usr/lib/jvm; tar xpzf $FILE
```

Comment:
I would usually use `/opt/jdk/` here as the archive contains binaries and libraries. 

You should also make sure that the path exists or is created before before running this command.